Dockerfile.deploy

```diff
@@ -77,35 +77,28 @@ COPY version.json /app/version.json
 COPY . /data/olympia
 WORKDIR /data/olympia
 
-# Install all python requires
+# Install all python requires, and add symlink for old uwsgi binary paths
```

Comment:
I'm not sure, removing them was a step too far for this patch - I just refactored the dockerfile purely because I had to build it a bunch of times so the lack of consolidation of the various stages jumped at me.